TrackingAntennaV3/main/Magnetometer.hpp

```diff
@@ -1,20 +1,23 @@
-/**
- * @file   Magnetometer.hpp
  * @Author Michael Lenover (WARG)
  * @date   January 5, 2017
  * @brief  Magnetometer initialization functions
+ *     Connection based off of the Arduino ethernet library.
+ *     Uses DHCP so no need to know the IP of the shield once connected.
```

Comment:
Edit hte comments plz. This is onl relevant for the ethernet stuff. Again you should clearly describe what this magnetometer does, how it connects, what pins you have to connect it to the arduino, etc...